v2.0/import.md

```diff
@@ -172,14 +172,17 @@ URLs for the file you want to import and your temp directory must use the follow
 | Google Cloud [<sup>1</sup>](#notes) | `gs` | Bucket name | None |
 | HTTP [<sup>2</sup>](#notes) | `http` | Remote host | N/A |
 | NFS/Local [<sup>3</sup>](#notes) | `nodelocal` | File system location | N/A |
+| S3-compatible services | `s3` | Bucket name | `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, `AWS_REGION`, `AWS_ENDPOINT` |
 
 #### Notes
 
 [<sup>1</sup>](#import-file-temp-directory-urls) Only supports instance auth.
 
 [<sup>2</sup>](#import-file-temp-directory-urls) You can easily create your own HTTP server with [Caddy or nginx](create-a-file-server.html).
 
-[<sup>3</sup>](#import-file-temp-directory-urls) If using NFS for your temp directory, each node in the cluster must have access to the NFS using the same URL.
+[<sup>3</sup>](#import-file-temp-directory-urls) If using NFS for your temp directory, each node in the cluster must have access to the NFS using the same URL. The file system backup location on the NFS drive is relative to the path specified by the `--external-io-dir` flag set while [starting the node](start-a-node.html). If the flag is set to `disabled`, then imports from local directories and NFS drives are disabled. 
```

Comment:
@dt Please review this section

Reply:
First sentence of this is no longer quite true (due to the second).
Now it is important only that each node in your cluster configure its `--external-io-dir` flag to point to its local mount point of the that NFS share (or, as an alternative, symlink from the default `extern` subdirectory to the NFS-backed directory).